src/utils/request-api.js

```diff
@@ -21,7 +21,7 @@ function parseError (res, cb) {
 
     if (payload) {
       error.code = payload.Code
-      error.message = payload.Message || payload.toString()
+      error.message = payload.message || payload.Message ||payload.toString()
       error.type = payload.Type
```

Comment:
Cluster will never set an error type, only `code` and `message`.

Reply:
updated to remove type